Makefile

```diff
@@ -19,8 +23,7 @@ NAME := netplugin
 VERSION_FILE := $(NAME)-version
 VERSION := `cat $(VERSION_FILE)`
 TAR_EXT := tar.bz2
-# BUILD_VERSION=1.2 make --> 1.2-1097d2a7, otherwise devbuild-1097d2a7
-export NETPLUGIN_CONTAINER_TAG := $(or $(BUILD_VERSION),devbuild)-$(shell ./scripts/getGitCommit.sh)
+NETPLUGIN_CONTAINER_TAG := $(shell ./scripts/getGitCommit.sh)
```

Comment:
IMO instead of using the script to determine the version, we can simply uses `$(git describe --tags --always)` to get the tag, the format will be 
`<latest-tag>-<commits-after-tag>-g<sha>`, if it's right on tag, it will be `<tag>`. this contains commit sha so should be enough to identify the change

Reply:
we should update the script (it's used in multiple places), that can be separate PR